### PR TITLE
scripts: remove github-merge.py

### PR DESCRIPTION
**Pull request by fanquake**
This script has been moved to the bitcoin-core/bitcoin-maintainer-tools repository,
after discussion in a core dev meeting. The rationale being that it is also useful to
other projects, and thus should be moved to a more general repository.

Meeting log: http://www.erisian.com.au/bitcoin-core-dev/log-2019-08-15.html
See also: https://github.com/bitcoin-core/bitcoin-maintainer-tools/pull/28

~~I'll open a follow up PR to port the documentation to the maintainer tools repo.~~
Ported the docs in https://github.com/bitcoin-core/bitcoin-maintainer-tools/pull/29

### CODE REVIEW

**Comment by Sjors** [comment]
Concept ACK. Also, I can't verify what script a maintainer runs anyway, can only verify the result. So imo it doesn't matter where this script lives.

**Comment by emilengler** [comment]
You should like to the new script in the other repository, for transparency.

**Comment by Empact** [comment]
https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/master/github-merge.py

File: contrib/devtools/README.md [hunk 1]
```diff
@@ -89,66 +89,6 @@ example:
 BUILDDIR=$PWD/build contrib/devtools/gen-manpages.sh
 ```
 
-github-merge.py
```
Comment (laanwj) [inline contrib/devtools/README.md]: Let's add this documentation to the maintainer tools repo too so it doesn't get lost.

File: contrib/README.md [hunk 2]
```diff
@@ -3,10 +3,9 @@ Repository Tools
 
 ### [Developer tools](/contrib/devtools) ###
 Specific tools for developers working on this repository.
-Contains the script `github-merge.py` for merging GitHub pull requests securely and signing them using GPG.
```
Comment (laanwj) [inline contrib/README.md]: Might make sense to mention the `maintainer-tools` repository (here, or somewhere else in a README), and that most developer tools live there.

File: contrib/devtools/README.md (hunk 1 above)
Comment (fanquake) [reply contrib/devtools/README.md]: I've got https://github.com/bitcoin-core/bitcoin-maintainer-tools/pull/29 open to do just that 👍

File: contrib/README.md (hunk 2 above)
Comment (fanquake) [reply contrib/README.md]: Agree, will update with a reference to the maintainer tools repo.

Comment (fanquake) [reply contrib/README.md]: Updated with a link to the maintainer-tools repository.

**Comment by laanwj** [comment]
ACK 6c27fa0f09e5ec3854f30673d26c6e5e2d27063e

File: contrib/README.md [hunk 3]
```diff
@@ -3,10 +3,10 @@ Repository Tools
...
+Additional tools, including the `github-merge.py` script, are available in the [maintainer-tools](https://github.com/bitcoin-core/bitcoin-maintainer-tools) repository.
 
 ### [Verify-Commits](/contrib/verify-commits) ###
-Tool to verify that every merge commit was signed by a developer using the above `github-merge.py` script.
+Tool to verify that every merge commit was signed by a developer using the `github-merge.py` script.
```
Comment (jnewbery) [inline contrib/README.md]: nit: s/signed by a developer/signed by a maintainer/

**Review by jnewbery — COMMENTED** [review]
one nit inline. Otherwise looks good.

**Pull request merged**